protobuf/mapping.go

```diff
@@ -17,12 +37,35 @@ func (t *ProtoType) Type() Type {
 	return NewNamed(t.Package, t.Name)
 }
 
+type Decorator func(*Package, *Message, *Field)
+type Decorators []Decorator
+
+func NewDecorators(fns ...func(*Package, *Message, *Field)) Decorators {
```

Comment:
You can use the created type:
`func NewDecorators(fns ...Decorator) Decorators {`